open-company/open-company-web
# Better bot removal

Main PR: https://github.com/open-company/open-company-auth/pull/92

The change to `src/oc/web/actions/user.cljs` is fixing an annoying bug where some times (usually on the first save) the web app was redirecting you to your first org in the list of available orgs after saving the user profile.
If you have only 1 org it's completely transparent but it's very annoying in case of multiple orgs (or teams) user.

NB: this is pretty simple and can be reviewed and deployed alone, but it's needed to test the main PR linked above. Following tests are not needed if this is reviewed with the other PRs.

To test:
- get a user with 2 orgs/teams: one is a Slack org/team with the bot added and the other is an email org
- go to the Slack org/team and open Notifications panel
- change all to Slack
- go to the email org-team
- open Notifications panel
- [ ] see Slack still selected?
- [ ] can you NOT select Slack from here though if it's not already selected?
- save your user profile
- wait 5 seconds or so
- [ ] did you NOT get redirected to your other org
